uiowa/uiowa
# Patch for node_revision_delete and content update for default_content

# How to test

1. composer install
2. blt ds --site=default.uiowa.edu
3. Try deleting an older revision of a page.
4. You should be able to delete 1 and all older revisions.

For the default content: 
1. Install a fresh default or sandbox site with blt drupal:install --site default or blt drupal:install --site sandbox.uiowa.edu.
2. Do you get the new homepage? See https://github.com/uiowa/uiowa/issues/4849#issuecomment-1080811154
3. Do you see the "Component Showcase" page in the main menu? It should look like the old default homepage. This was done to preserve the old homepage.

General editorial check/comparison between https://sandbox.prod.drupal.uiowa.edu/homepage-example-4 and the new homepage.

Will also address #4849 and #4977
